Cargo.toml

```diff
@@ -59,4 +59,4 @@ xdg = "2.4"
 
 
 [patch.crates-io]
-bpaf = { git = "https://github.com/ysndr/bpaf", branch = "error/more-suggestions" }
+# bpaf = { git = "https://github.com/ysndr/bpaf", branch = "error/more-suggestions" }
```

Comment:
Can we drop the patch section entirely now?

Reply:
yeah, I reported a bug to be published on crates "later today" until then we need the override for a bit longer, thereafter we can remove the patch, yes